deepspeed/checkpoint/ds_to_universal.py

```diff
@@ -70,6 +75,19 @@ def parse_arguments():
     return args
 
 
+def atoi(text):
+    return int(text) if text.isdigit() else text
+
+
+def natural_keys(text):
```

Comment:
Thank you for introducing this interesting approach.
We have a similar sorting in `_merge_zero_shards` but are using a different approach but it is not good to have two different sorting implementations for the same purpose. Can you replace this one with `natural_keys`?

Reply:
Thanks for the suggestions!
For this `natural_keys` function, I actually reuse it from [zero_to_fp32.py](https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/utils/zero_to_fp32.py#L52).

You’re right; it’s not ideal to have two different implementations for the same function. How about I replace the one in `_merge_zero_shards` with this `natural_keys`?

Reply:
This sounds good, thank you